### PR TITLE
Update dynaflow parameters

### PR DESCRIPTION
**Pull request by dimbdr**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
The Dynaflow Launcher team has requested to not export Dynaflow parameters if they're not set by a user or the configuration platform.
By default, there's no exported parameter.

**What is the current behavior?** *(You can also link to an open issue here)*
All the Dynaflow parameters have a default value, and are all exported when serialized in JSON format.

**What is the new behavior (if this is a feature change)?**
If a parameter is set by the user or by the platform configuration, then this parameter wiill be exported to a serialized JSON format.
Otherwise, it will be ignored.

**Other information**:
In addition, a commit that deletes 2 parameters, as requested : VscAdGenerators and LccAsLoads

### CODE REVIEW

File: dynaflow/src/test/java/com/powsybl/dynaflow/DynaFlowParametersTest.java [hunk 1]
```diff
@@ -149,25 +246,50 @@ public void parametersSerialization() throws IOException {
...
-        assertFalse(dynaFlowParameters.getLccAsLoads());
-        assert 0 == dynaFlowParameters.getTimeStep();
+        assert dsoVoltageLevel == dynaFlowParameters.getDsoVoltageLevel();
+        assertEquals(activePowerCompensation, dynaFlowParameters.getActivePowerCompensation());
+        assertEquals(settingPath, dynaFlowParameters.getSettingPath());
+        assertEquals(assemblingPath, dynaFlowParameters.getAssemblingPath());
+        assert startTime == dynaFlowParameters.getStartTime();
+        assert stopTime == dynaFlowParameters.getStopTime();
+        assert precision == dynaFlowParameters.getPrecision();
+        //assert timeOfEvent == dynaFlowParameters.getTimeOfEvent();
```
Comment (marcochiaramello) [inline dynaflow/src/test/java/com/powsybl/dynaflow/DynaFlowParametersTest.java:291]: could you remove this commented line? thx

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: dynaflow/src/test/java/com/powsybl/dynaflow/DynaFlowParametersTest.java (hunk 1 above)
Comment (dimbdr) [reply dynaflow/src/test/java/com/powsybl/dynaflow/DynaFlowParametersTest.java]: Ups
Done

**Review by marcochiaramello — APPROVED** [review]

File: dynaflow/src/test/java/com/powsybl/dynaflow/json/JsonDynaFlowParametersSerializerTest.java [hunk 2]
```diff
@@ -35,8 +48,16 @@ public void testDeserialize() {
         assertTrue(dynaFlowParameters.getSvcRegulationOn());
         assertFalse(dynaFlowParameters.getShuntRegulationOn());
         assertTrue(dynaFlowParameters.getAutomaticSlackBusOn());
-        double expectedDsoVoltageLevelValue = 987.6;
         assertEquals(expectedDsoVoltageLevelValue, dynaFlowParameters.getDsoVoltageLevel(), 0);
+        assertEquals(DynaFlowConstants.ActivePowerCompensation.P, dynaFlowParameters.getActivePowerCompensation());
+        assertEquals(expectedSettingPath, dynaFlowParameters.getSettingPath());
+        assertEquals(expectedAssemblingPath, dynaFlowParameters.getAssemblingPath());
+        assert expectedStartTime == dynaFlowParameters.getStartTime();
```
[the remark this answers is not in the document]
Comment (dimbdr) [reply dynaflow/src/test/java/com/powsybl/dynaflow/json/JsonDynaFlowParametersSerializerTest.java]: Corrected

**Review by gautierbureau — APPROVED** [review]

**Review by flo-dup — CHANGES_REQUESTED** [review]
Could you remove the `Objects.nonNull` and `Objects.isNull`?

File: dynaflow/src/main/java/com/powsybl/dynaflow/DynaFlowParameters.java [hunk 3]
```diff
@@ -6,65 +6,93 @@
...
+    private static final String SVC_REGULATION_ON = "svcRegulationOn";
+    private static final String SHUNT_REGULATION_ON = "shuntRegulationOn";
+    private static final String AUTOMATIC_SLACK_BUS_ON = "automaticSlackBusOn";
+    private static final String DSO_VOLTAGE_LEVEL = "dsoVoltageLevel";
+    private static final String ACTIVE_POWER_COMPENSATION = "activePowerCompensation";
+    private static final String SETTING_PATH = "settingPath";
+    private static final String ASSEMBLING_PATH = "assemblingPath";
+    private static final String START_TIME = "startTime";
+    private static final String STOP_TIME = "stopTime";
+    private static final String PRECISION = "precision";
```
Comment (flo-dup) [inline dynaflow/src/main/java/com/powsybl/dynaflow/DynaFlowParameters.java:77]: You have a clash between `PRECISION` static variable and `precision` variable. Could you rename one of them?

Comment (dimbdr) [reply dynaflow/src/main/java/com/powsybl/dynaflow/DynaFlowParameters.java]: Done

**Review by flo-dup — APPROVED** [review]

**Pull request merged**